sql/plsql/PlSqlLexer.g4

```diff
@@ -418,6 +418,7 @@ DELETEXML                      : 'DELETEXML';
 DEMAND                         : 'DEMAND';
 DENSE_RANKM                    : 'DENSE_RANKM';
 DEPENDENT                      : 'DEPENDENT';
+DEPRECATE                      : 'DEPRECATE';
```

Comment:
I guess it can be an identifier. If so, I suggest adding it to [`regular_id`](https://github.com/antlr/grammars-v4/blob/master/sql/plsql/PlSqlParser.g4#L7442C1-L7442C11) (or `non_reserved_keywords_pre12c`, `non_reserved_keywords_in_12c`, `non_reserved_keywords_in_18c`).

Reply:
Amended commit https://github.com/antlr/grammars-v4/pull/4012/commits/d875c67b00330d87aa0b15f335d39a1b18ee1631